testdata/deponixml/noark5-enkel/arkivstruktur.xml

```diff
@@ -53,9 +52,11 @@
         <telefonnummer>22221122</telefonnummer>
         <kontaktperson>Alice</kontaktperson>
       </part>
+      <!--
       <kryssreferanse>
```

Comment:
For some reason adding kryssreferanse to the "excludes" did not work, so I commented it out.